.tekton/integration-tests/coo-e2e-test-fbc-ocp-ci-job-trigger-pipeline.yaml

```diff
@@ -0,0 +1,168 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  annotations:
+    pipelinesascode.tekton.dev/task: "[coo-e2e-tests]"
+  name: coo-e2e-test-fbc
+spec:
+  description: |
+    This pipeline automates the process of running e2e tests for Cluster Observability Operator. The pipeline fetches the FBC fragment image and
+    passes it as a parameter to the OpenShift CI job using Gangway API.
+  params:
+    - name: SNAPSHOT
+      description: 'The JSON string representing the snapshot of the application under test.'
+      default: '{"components": [{"name":"test-app", "containerImage": "quay.io/example/repo:latest"}]}'
```

Comment:
Not sure we should leave this default? If this gets used accidentally we might get a false test signal.